start-a-local-cluster.md

```diff
@@ -6,21 +6,21 @@ toc: false
 Once you've [installed CockroachDB locally](/install-cockroachdb), the quickest way to test out the database is to start a single-node cluster and talk to the node via the built-in SQL client. There are two ways to do this:
 
 - [Development Mode (Insecure)](#development-mode)  
-Data is stored in-memory and client/server communication is completely insecure. This mode is great for learning CockroachDB functionality but not suitable for actual development.
+Data is stored in-memory and client/server communication is completely insecure. This mode is great for learning CockroachDB, but there's no authentication or encryption, so it's not suitable for actual development.
```

Comment:
And it's not that it's not suited for actual development. Better to say, "so it's suitable only for limited testing and development."